pkg/network/usm/postgres_monitor_test.go

```diff
@@ -9,13 +9,15 @@ package usm
 
 import (
 	"fmt"
+	"github.com/stretchr/testify/assert"
```

Comment:
please move to the correct location

Reply:
Guy, it is "go fmt" which orders imports automatically

Reply:
go fmt organizes each group by lexicographic order
you need to manually split it to groups with an empty line between the groups

Reply:
done